aspects/schema.go

```diff
@@ -143,6 +144,15 @@ func (s *StorageSchema) Validate(raw []byte) error {
 	return s.topLevel.Validate(raw)
 }
 
+// AtPath returns the types that may be stored at the specified path.
+func (s *StorageSchema) AtPath(path []string) ([]Schema, error) {
+	return s.topLevel.AtPath(path)
+}
+
+func (s *StorageSchema) Type() SchemaType {
+	return Invalid
```

Comment:
shouldn't this just be s.toplevel.Type() as well?

Reply:
It can but like alternativesSchema, an AtPath should really land here as this is just a wrapper around other elements

Reply:
we made Type separate from TypeAtPath, so it needs to make sense more generally, and to me it sounds odd to declare that the wrapper schema is Invalid, it's only "invalid" from a very specific pov, but not looking at the API as a whole from outside.